ktest/runtests.sh

```diff
@@ -73,6 +73,18 @@ installertest()
 	#../src/pltcalc/pltcalc -M ../ktest/testout/fmmplt1.csv < ../ktest/testout/fmsummarycalc1.bin 
 	#../src/pltcalc/pltcalc -M ../ktest/testout/fmmplt2.csv < ../ktest/testout/fmsummarycalc2.bin
 
+	# test qelt
+	../src/eltcalc/eltcalc -Q ../ktest/testout/gulqelt1.csv < ../ktest/testout/gulsummarycalc1.bin 
+	../src/eltcalc/eltcalc -Q ../ktest/testout/gulqelt2.csv < ../ktest/testout/gulsummarycalc2.bin 
+	#../src/eltcalc/eltcalc -Q ../ktest/testout/fmmelt1.csv < ../ktest/testout/fmsummarycalc1.bin 
+	#../src/eltcalc/eltcalc -Q ../ktest/testout/fmmelt2.csv < ../ktest/testout/fmsummarycalc2.bin 
+
+	# test qplt
+	../src/pltcalc/pltcalc -Q ../ktest/testout/gulqplt1.csv < ../ktest/testout/gulsummarycalc1.bin 
+	../src/pltcalc/pltcalc -Q ../ktest/testout/gulqplt2.csv < ../ktest/testout/gulsummarycalc2.bin 
+	#../src/pltcalc/pltcalc -Q ../ktest/testout/fmmplt1.csv < ../ktest/testout/fmsummarycalc1.bin 
+	#../src/pltcalc/pltcalc -Q ../ktest/testout/fmmplt2.csv < ../ktest/testout/fmsummarycalc2.bin
+
```

Comment:
It is possible to output the QELT and MELT (or QPLT and MPLT) in a single run of `eltcalc` (or `pltcalc`). So, the following would also be valid:

```
../src/eltcalc/eltcalc -M ../ktest/testout/gulmelt1.csv -Q ../ktest/testout/gulqelt1.csv < ../ktest/testout/gulsummarycalc1.bin
../src/pltcalc/pltcalc -M ../ktest/testout/gulmplt1.csv -Q ../ktest/testout/gulqplt1.csv < ../ktest/testout/gulsummarycalc1.bin
```

Reply:
sure we can make that change